components/system/components/Slider.js

```diff
@@ -0,0 +1,242 @@
+import * as React from "react";
+import * as Constants from "~/common/constants";
+
+import { css } from "@emotion/react";
+import { DescriptionGroup } from "~/components/system/components/fragments/DescriptionGroup";
+import { Input } from "~/components/system/components/Input";
+
+import Draggable from "react-draggable";
+
+const STYLES_BAR_CONTAINER = css`
+  height: 48px;
+  display: grid;
+  grid-template-columns: 1fr auto;
+  align-items: center;
+`;
+
+const STYLES_SLIDER_BAR = css`
+  width: calc(100% - 24px);
```

Comment:
Why do you need to subtract 24px?

Reply:
I have it so when the handle is at the far left or right, the bar end is in the middle of the dot. That way it can fully reach all the values along the whole slider. If I had the bar extend the whole length, measurements based off of the width of the bar (which are used to calculate the returned `value`) would be off.

Reply:
Interesting, clever idea! I'm going to do a scratch to see if I can simplify this and show you at a later point.